src/pages/Stake/Active/index.tsx

```diff
@@ -22,16 +22,16 @@ import { GenerateNominations } from '../GenerateNominations';
 import { PageTitle } from '../../../library/PageTitle';
 import { OpenAssistantIcon } from '../../../library/OpenAssistantIcon';
 import { useModal } from '../../../contexts/Modal';
-import StatBoxListItem from '../../../library/StatBoxList/Item';
-import { useStats } from './stats';
 import { PAYEE_STATUS } from '../../../constants';
+import ActiveNominationsStatBox from './Stats/ActiveNominations';
+import MinimumActiveBondStatBox from './Stats/MinimumActiveBond';
+import ActiveEraStatBox from '../../Validators/Stats/ActiveEra';
```

Comment:
Good workaround. I think the ActiveEra is more suited to the staking page than Validators.

Perhaps we can introduce another stat to the validator page, maybe how many validators are validating parachains or something along those lines.